pkg/emb6/patches/0002-Rename-colliding-files-and-functions.patch

```diff
@@ -1,39 +1,40 @@
-From 4a74cf7e617a7e684dc0a5b5534e26f9d3cb5817 Mon Sep 17 00:00:00 2001
+From 4e6d1640adde2828f10320d39f8996c6e89f82ac Mon Sep 17 00:00:00 2001
 From: Martine Lenders <mail@martine-lenders.eu>
 Date: Thu, 4 Feb 2016 22:15:23 +0100
-Subject: [PATCH 2/4] Rename colliding files and functions
+Subject: [PATCH 2/2] Rename colliding files and functions
```

Comment:
:-/ I usually try to `git format-patch` the whole patch collection so this doesn't happen.

Reply:
But patch number 3 got removed at some point

Reply:
Want me to add a commit changing the patches to fix the numbering?

Reply:
Ah. Well then forget what I said :D

Reply:
> Want me to add a commit changing the patches to fix the numbering?

That's optional, but I wouldn't be mad about it ;-)

Reply:
Done